metricbeat/docs/modules/aerospike/namespace.asciidoc

```diff
@@ -7,6 +7,7 @@ This file is generated! See scripts/mage/docs_collector.go
 
 include::../../../module/aerospike/namespace/_meta/docs.asciidoc[]
 
+This is a default metricset.
```

Comment:
Will users know what we mean when we say that it is a default metricset? I wonder if it might be better to say something like "This metricset is enabled by default."

Reply:
Yah, I was debating about the language here. In this case, _default_ means "This metricset will be enabled if the module is not configured" which seems...subtly different? Maybe?

Reply:
I think there's still some ambiguity though. Precise and concise are always at odds. :-)

Let's just leave it as you have it for now (This is a default metricset) and see if users are confused by the language.

Reply:
We could also link to the page that explains what a default module is too.